packages/data/src/index.js

```diff
@@ -15,9 +15,17 @@ export { default as withRegistry } from './components/with-registry';
 export { default as RegistryProvider, RegistryConsumer } from './components/registry-provider';
 export { default as __experimentalAsyncModeProvider } from './components/async-mode-provider';
 export { createRegistry } from './registry';
-export { plugins };
 export { createRegistrySelector, createRegistryControl } from './factory';
 
+/**
```

Comment:
I wish we had a better way to auto-document this. Unfortunately, we don't yet. 

- One approach could've been using `@property` tags. Current issue https://github.com/WordPress/gutenberg/issues/15186
- Another approach could've been using whatever mechanism we end up implementing to group things together. Current issue: https://github.com/WordPress/gutenberg/issues/15178